core/test/functional/routes/api/notifications_spec.js

```diff
@@ -50,6 +50,43 @@ describe('Notifications API', function () {
                     jsonResponse.notifications[0].type.should.equal(newNotification.type);
                     jsonResponse.notifications[0].message.should.equal(newNotification.message);
                     jsonResponse.notifications[0].status.should.equal('alert');
+                    jsonResponse.notifications[0].dismissible.should.be.true();
+                    should.exist(jsonResponse.notifications[0].location);
+                    jsonResponse.notifications[0].location.should.equal('bottom');
+                    jsonResponse.notifications[0].id.should.be.a.String();
+
+                    done();
+                });
+        });
+
+        xit('should not be able to set custom id from outside', function (done) {
```

Comment:
@kirrg001 this clearly seems like a bug based on the test case the had been removed? should we open up an issue or the initial test was asserting a wrong thing from the beginning? :thinking: